docs/omero-release-process.rst

```diff
@@ -0,0 +1,108 @@
+ - Add an entry to `history.rst <https://github.com/ome/openmicroscopy/blob/develop/history.rst>`_.
+
+After committing the changes, a signed tag must be created for the released version
+using :command:`git tag -s`::
+
+    $ git tag -s -m "Tag version x.y.z" vx.y.z
+
+Both the `develop` branch as well as the tag must be pushed upstream::
+
+    $ git push origin develop vx.y.z
+
+This will trigger GitHub action builds, generate artifacts, create a GitHub release and upload
+the artifacts to the release on GitHub.
+An hourly cron job runs on our virtual machine and copy the artifacts published on Github under `OMERO Downloads`_.
+
+Docker container release
+------------------------
+
+Before making a new release, merge all contributions on the `master` branch (rare).
+
+When the artifacts are available under `OMERO Downloads`_, you can release the `OMERO Server Docker`_.
+Bump the version in `Dockerfile <https://github.com/ome/omero-server-docker/blob/master/Dockerfile>`_ to the latest release version.
+
+After committing the changes, a signed tag must be created for the released version
+using :command:`git tag -s`::
+
+    $ git tag -s -m "Tag version x.y.z" x.y.z
+
+Both the `master` branch as well as the tag must be pushed upstream::
+
+    $ git push origin master x.y.z
+
+
+Documentation release
+---------------------
+
+Before making a new release, merge all contributions on the `develop` branch.
+
+A GitHub action is run daily to update the documentation with changes made in various repositories
+e.g. ``omero-install``, ``omeroweb-install``, ``openmicroscopy``, and bump the version of various components e.g. ``omero-web``.
+If you wish to publish the documentation the same day as the source code release, the easiest option is to 
+modify the format of the cron job in the `update.yml <https://github.com/ome/omero-documentation/blob/develop/.github/workflows/update.yml>`_ GitHub action, commit and push the change to the `develop` branch.
+
+After a succesful run of the `update.yml <https://github.com/ome/omero-documentation/blob/develop/.github/workflows/update.yml>`_ GitHub action, merge the Pull Request created by the action if changes were made in the related repositories.
+After fetching the `develop` branch, revert the change to the cron job in :file:`update.yml`. A signed tag must be created for the released version using :command:`git tag -s`::
+
+    $ git tag -s -m "Tag version x.y.z" vx.y.z
+
+Both the `develop` branch as well as the tag must be pushed upstream::
+
+    $ git push origin develop vx.y.z
+
+The documentation is then published on `Read The Docs`_.
```

Comment:
See the build progress on the `documentation builds <https://readthedocs.org/projects/omero/builds/>`_ page.